main/src/main/java/cgeo/geocaching/ui/TextParam.java

```diff
@@ -118,6 +119,14 @@ public TextParam setImage(final ImageParam image, final int imageSizeInDp) {
         return this;
     }
 
+    /**
+     * set tint color for compound image
+     */
+    public TextParam setImageTint(final int imageTintColor) {
```

Comment:
I assume this is a color int? Then you should annotate the parameter with `@ColorInt`

Reply:
thx, have added the annotation